notebooks/run_model_in_notebook.ipynb

```diff
@@ -15,7 +15,8 @@
    "outputs": [],
```

Comment:
<p><code>b'Reading config from /home/sarah/temp/ecoextreml/test/input/ZA-Kru_2022-08-03-1537/ZA-Kru_2022-08-03-1537_config.txt<strong>\n\n</strong> The calculations start now \r\n The calculations end now \r'</code></p><p> Not the very inconsistent line breaks here;</p><p><code>\n\n</code></p><p><code>\r\n</code></p><p>and <code>\r</code></p><p>I think this should be changed in STEMMUS_SCOPE.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
good catch, I fix it [here](https://github.com/EcoExtreML/STEMMUS_SCOPE/pull/105/commits/13cc166a499830a154ab58114b8a1c3803374b47) .